draft-ietf-tls-external-psk-guidance.md

```diff
@@ -416,16 +431,21 @@ the application will be given precedence over how to handle the PSK.
 
 # Security Considerations {#security-con}
 
+Security considerations are provided throughout this document.  It bears
+repeating that there are concerns related to the use of external PSKs related
+to proper identification of the TLS version 1.3 endpoints and additional
+risks when the external PSKs are know to a group.
+
 It is NOT RECOMMENDED to share the same PSK between more than one client and server.
 However, as discussed in {{use-cases}}, there are application scenarios that may
-rely on sharing the same PSK among multiple nodes. {{!I-D.ietf-tls-external-psk-importer}}
+rely on sharing the same PSK among multiple nodes. {{I-D.ietf-tls-external-psk-importer}}
 helps in mitigating rerouting and Selfie style reflection attacks when the PSK
 is shared among multiple nodes. This is achieved by correctly using the node
 identifiers in the ImportedIdentity.context construct specified in
-{{!I-D.ietf-tls-external-psk-importer}}. It is RECOMMENDED that each endpoint
+{{I-D.ietf-tls-external-psk-importer}}. It is RECOMMENDED that each endpoint
 selects one globally unique identifier and uses it in all PSK handshakes. The
```

Comment:
Apparently I didn't find this noteworthy the first time around, but reading it again, "RECOMMENDED to always use a single fixed GUID" has pretty lousy privacy properties.  Even if we think this is the right tradeoff based on the functionality needs, maybe we should acknowledge that it has poor privacy properties.

Reply:
Couldn't hurt! Good suggestion.

Reply:
s/It is RECOMMENDED that each/One solution is that/

Reply:
Yep, that'd work.